ebos/eclmpiserializer.hh

```diff
@@ -48,6 +48,32 @@ public:
             Mpi::unpack(const_cast<T&>(data), m_buffer, m_position, m_comm);
     }
 
+    template<class T>
+    void vector(const std::vector<T>& datac)
+    {
+        std::vector<T>& data = const_cast<std::vector<T>&>(datac);
+
+        auto handle = [&](auto& d)
+        {
+            for (auto& it : d) {
```

Comment:
SO this is never called for a vector of pods? Shouldn't there be at least a check (e.g. `static_assert<!std::is_pod<T>::values, "No no no">()`?

Reply:
Well I guess that would yield a prettier error message. You'd get no member serializeOp if you try now.